node-fs-writefile/note-to-self.js

```diff
@@ -0,0 +1,23 @@
+const fs = require('fs');
+const note = process.argv[2] + '\n';
+const writeFileButBetter = (data, encoder) => {
+  fs.writeFile('note.txt', data, encoder, err => {
+    if (err) throw err;
+  });
+};
+
+fs.readFile('note.txt', 'utf8', (err, data) => {
+  try {
+    writeFileButBetter('');
+  } catch (error) {
+    throw err;
+  }
+
+  switch (data) {
```

Comment:
This looks like an attempt to append the note? This is a nice approach, though syntactically, using `switch` here is weird since it only has one non-default case.

A better approach would be to use appendFile instead of writeFile.